.ci/oci-e2e-deployment.sh

```diff
@@ -15,13 +15,12 @@ export MY_GITHUB_ORG="redhat-appstudio-qe"
 export MY_GITHUB_TOKEN="${GITHUB_TOKEN}"
 export E2E_APPLICATIONS_NAMESPACE=appstudio-e2e-test
 
+export PATH=$PATH:/tmp/bin
+mkdir -p /tmp/bin
+
```

Comment:
@flacatus could you please add this env var to the script
```suggestion
# Environment variable used to override the default "protected" image repository in HAS
# https://github.com/redhat-appstudio/application-service/blob/6b9d21b8f835263b2e92f1e9343a1453caa2e561/gitops/generate_build.go#L50
# Users are allowed to push images to this repo only in case the image contains a tag that consists of "<USER'S_NAMESPACE_NAME>-<CUSTOM-TAG>"
# For example: "quay.io/redhat-appstudio-qe/test-images-protected:appstudio-e2e-test-mytag123"
export HAS_DEFAULT_IMAGE_REPOSITORY="quay.io/redhat-appstudio-qe/test-images-protected"
```

this should solve the issue with build-service-suite:
```
[FAIL] [build-service-suite Build Service E2E tests] when creating a component with a specific container image URL [It] should fail for ContainerImage field set to a protected repository (without an image tag) [image repository protection]
/go/src/github.com/redhat-appstudio/infra-deployments/tmp/e2e-tests/tests/build/build.go:425
```